src/facebook/facebook-notifications/facebooknotificationsyncadaptor.cpp

```diff
@@ -64,6 +66,9 @@ void FacebookNotificationSyncAdaptor::finalize(int accountId)
         m_db.purgeOldNotifications(OLD_NOTIFICATION_LIMIT_IN_DAYS);
         m_db.sync();
         m_db.wait();
+
+        // manage image cache
+        purgeCachedImages(&m_imageCacheDb, accountId, QDateTime::currentDateTime().addDays(14 * -1));
```

Comment:
Propbably a good idea as we may want to adjust this for different account types. Two weeks started to feel a bit too short (there was no science behind it, it was just the first number that came to mind). There are two types of images we want to cache: avatars and feed content. Feed content is kind of a oneshot, it appears on top of the feed, is pushed down (in case of Twitter probably quite quickly) and is gone for good, just waiting to be cleared. Avartars on the other hand are used over and over again. I'm thinking 3 or 4 weeks would be a good compromise. If we relaod avatar images once per month, that's not a big deal.